aredis/client.py

```diff
@@ -1,9 +1,6 @@
 import asyncio
 import sys
-try:
-    from asyncio import CancelledError
-except ImportError:
-    from asyncio.futures import CancelledError
```

Comment:
try catch here to compat lower python version

Reply:
Sure we can keep that code as is. I'm not sure of which low python version needs that.

Reply:
See issue #122